dbcon/joblist/tuple-bps.cpp

```diff
@@ -1933,7 +1934,8 @@ void TupleBPS::receiveMultiPrimitiveMessages(uint32_t threadID)
     bool didEOF = false;
     bool unused;
 
-    /* Join vars */
+    // Join vars
+    // These can't be members of tupleBPS because this* may be used by multiple threads
```

Comment:
Denis rewrote the TBPS::RMPM() so now there is a very limited part of TBPS::RMPM()  logic where multiple threads are used.

Reply:
And they don't share crit section execution in the most heavy part.

Reply:
When we merge the two, I will see if we can reduce the arg count. It is excessive.